vispy/app/tests/test_ipython.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""Tests for Vispy's IPython bindings"""
+
+# from vispy import IPython
+from numpy.testing import assert_equal
+from IPython.testing.globalipapp import get_ipython
```

Comment:
nest this import, it will fail on systems without IPython, defeating the purpose of the decorator (Travis will also tell you this once we put the `ipython` install only on the `DEPS=full` case